source/agora/consensus/EnrollmentManager.d

```diff
@@ -966,26 +951,26 @@ public class EnrollmentManager
 
     /***************************************************************************
 
-        Set the key for the enrollment for this node
+        Set the UTXO for the enrollment for this node
 
-        If saving an enrollment key fails, this displays a log messages and just
+        If saving to the database fails, this displays a log messages and just
         returns. The `enroll_key` stores the value anyway and it can be restored
```

Comment:
Ah, There's one more to be fixed.

Reply:
No, this one references the field in `EnrollmentManager` which has the same name.